Sources/StreamChatUI/Utils/UIStoryboard+Extensions.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+// swiftlint:disable force_cast
+extension UIStoryboard {
```

Comment:
Can be removed since we don't leverage storyboards in the SDKs

Reply:
Good to keep for the future?

Reply:
it decreases the test coverage and we don't have plans to leverage storyboards internally. i'd remove but leaving it up to you